google_analytics/notebooks/aggregate_per_customer.ipynb

```diff
@@ -163,21 +165,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
```

Comment:
Time to upgrade, buddy;)

Reply:
Backwards compatibility. You know for our huge user base of 6 people, none of which runs Python 2 anyway